src/apps/intel_mp/test_10g_vmdq_pool_sel.sh

```diff
@@ -0,0 +1,69 @@
+#!../../snabb snsh
```

Comment:
It turns out that the original `intel_mp` code uses the `.snabb` extension for files that run with Lua, so I renamed the additional tests to match them (at least then it will get no highlights instead of wrong ones---I'm also ok with switching all of them to `.lua` too).